### DISCUSSION TITLE
Paramètres mention "* Réponse obligatoire"

### DISCUSSION THREAD

**Discussion by jodionrq** [MTESSDev/FRW#31] [Annonces]
Bonjour,
Est-ce qu'il y a moyen de configurer nos formulaires pour changer le texte de la mention "* Réponse obligatoire" qui apparait en haut et à droite de chaque page sous le titre de la section? Ex.: si on veut écrire plutôt "Les champs obligatoires sont indiqués par le symbole *" ?

**Comment by j0ber** [comment]
Bonjour,

Non ce n'est pas possible et il ne faut pas. Aucune raison que ce texte soit différent d'un formulaire à l'autre.

Le texte doit être identique pour tous.

Cela dit s'il y a consensus sur un autre libellé nous sommes ouverts, mais il devrait rester court.

Le texte que vous suggérez est inutilement long et sur un téléphone il sortirait sur 2 lignes! (évidemment, l'astérique serait positionné différemment que sur cette image, mais ca donne une idée)
![image](https://user-images.githubusercontent.com/26974817/225986793-9813dcfd-4372-4b6a-952d-29713c0a8542.png)

De plus, ce texte n'est pas lu au lecteur écran. Il peut donc être court. Les utilisateurs connaissent déjà l'astérisque rouge.

**Reply by jodionrq** [reply]
Merci pour la réponse, bonne journée!
